### PR TITLE
Clarified the BrowserGlobalsCodeBlock in the GettingStardedPage with …

### PR DESCRIPTION
**Pull request by Sporego**
### Code block was confusing and could be more informative
*Used React.js Documentation comment style
*Clarified React.js Dependency `<script/>` tag
*Changed last script tag to `your code here` style instead of being ambiguous

### CODE REVIEW

**Comment by jquense** [comment]
Thanks, I think we'll keep it simple and not have extra comments.

**Comment by Sporego** [comment]
@jquense It is not simple and unclear because: 
1. If you follow the default, main, or recommended React tutorials @ [reactjs.org](https://reactjs.org/docs/create-a-new-react app.html#recommended-toolchains) you would have used React via create-react-app or some other recommended toolchain (what Facebook calls react framework).
    * Any unexperienced or new react dev would probably just copy and paste the whole code block... find that things are working and have unnecessary code in their project or worse get confused
1. `<script>var Alert = ReactBootstrap.Alert;</script>` can be `/*Your other script tags*/`. This is obviously clearer and more direct then the current example which for whatever reason references ReactBootsrap...

    Honestly, I'm a little butthurt because I was super excited to fork>commit>make a pull request. However, I understand why you may think my method of clarifying (e.g., too wordy, untasteful, "insert dissenting opinion 3" ) the documentation issues stated above isn't simple. I'm totally cool with that. 

My hope is you understand the issues stated above and solve it in whatever way you please. May your solution be ever clearer and simpler! 

Thanks and Best Wishes,
